lib/model/model.go

```diff
@@ -1964,8 +1969,10 @@ func (s *indexSender) Stop() {
 	<-s.stopped
 }
 
-// Complete always returns true, as Serve only terminates when a connection is
-// closed/has failed, in which case retrying doesn't help.
+// Complete returning true means that the suture supervisor will not restart
+// this service if it wasn't stopped. Here it always returns true, as indexSender
+// only terminates when a connection is closed/has failed, in which case
+// retrying doesn't help.
```

Comment:
We should explain that there is a hidden interface satisfaction check in suture for this.

The other option would be for the folder to be supervisor where all of its cruft gets run (versiomer, index senders et al), so we tear down the whole supervisor.

Also, it's a great opportunity to move this thing into a separate file.